src/bidiMapper/domains/network/NetworkRequest.ts

```diff
@@ -210,6 +211,29 @@ export class NetworkRequest {
       requestId: fetchRequestId,
       errorReason,
     });
+
+    // TODO: mark is as non-blocked?
+    // this.#isBlocked = false;
+  }
+
+  /** @see https://chromedevtools.github.io/devtools-protocol/tot/Fetch/#method-continueRequest */
+  async continueRequest(
+    fetchRequestId: Protocol.Fetch.RequestId,
+    url?: string,
+    method?: string
+  ) {
+    // TODO: Expand.
+    await this.#cdpTarget.cdpClient.sendCommand('Fetch.continueRequest', {
```

Comment:
What is the request is not blocked? Should it be handled in this method?

Reply:
I don't understand the comment. Could you elaborate?

Reply:
What if user tries to continue the request which is not blocked?

Reply:
Added `test_continue_request_non_blocked_request`